src/ActionModal/components/ActionModal.vue

```diff
@@ -438,7 +443,9 @@ export default {
     },
     signMethods() {
       let signMethods = []
-      if (this.session.sessionType === sessionType.EXPLORE) {
+      if (this.isMobileApp) {
```

Comment:
is there a explore session on mobile? in that case, what would the sign page show?

Reply:
yes there is. good question! i guess it should show a link to creating an account or signing in.